### DISCUSSION TITLE
The xyz plot script in forge does not support controlnet, when will it be fixed?

### DISCUSSION THREAD

**Discussion by pjwaixingren** [lllyasviel/stable-diffusion-webui-forge#831] [Q&A]
The xyz plot script in forge does not support controlnet, when will it be fixed?

**Comment by ArnorWing** [comment]
On December 15th at 13:41.

**Reply by hoodady** [reply]
i think its actually 13:47

**Comment by pjwaixingren** [comment]
<img width="1453" height="742" alt="image" src="https://github.com/user-attachments/assets/5dcb1eeb-fa25-42e7-ac89-9d012b453d34" /> YES  ,It`s cool!!it`s working...
